Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/travisci/travis-node-js:latest
+FROM travisci/ci-nodejs:packer-1494866191
```

Comment:
Where's this version number coming from? Is it just so that we point to a specific version, and don't get bitten by random updates?

Reply:
if a tag isn't provided docker looks for "latest", i guess that's common for docker image naming. but the travis images don't have a latest so i had to specify the latest tag manually